src/FindPackageTitlePlugin.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import noop from 'lodash/noop';
+
+import {
+  cleanup,
+  render,
+} from '@testing-library/react';
+
+import FindPackageTitlePlugin from './FindPackageTitlePlugin';
+
+jest.mock('./components', () => ({
+  SearchModal: jest.fn(() => <span>SearchModal</span>),
+}));
+
+const renderFindPackageTitlePlugin = renderCustomTrigger => render(
```

Comment:
Wouldn't be better to wrap single argument into brackets too? (according to the https://github.com/airbnb/javascript#arrows--paren-wrap)
```suggestion
const renderFindPackageTitlePlugin = (renderCustomTrigger) => render(
```

Reply:
I'm not sure, because if it was as it, we would use this rule earlier. Actually, this point needs to be discussed, Maybe it will be better to add this rule to omit the same comments in future, but I like this, that the single arguments without brackets, as for me, code looks cleaner.
@BogdanDenis what did you think about it?

Reply:
Let's wrap single parameter in arrow functions that are more than one line of code